docs/usage.md

```diff
@@ -158,11 +158,22 @@ are also required:
 -I $IA2_PATH/runtime/libia2/include
 -I $IA2_PATH/runtime/partition-alloc/include
 -Werror=incompatible-pointer-types
--Wl,--wrap=pthread_create
--Wl,--wrap=calloc
--Wl,--wrap=malloc
--Wl,--wrap=realloc
--Wl,--wrap=free
+-Wl,--wrap,pthread_create
```

Comment:
each `-Wl,--wrap` must be followed by `=` not `,`

Reply:
Ah noted, that should be fixed now.